src/Tribe/Rewrite.php

```diff
@@ -506,6 +516,13 @@ protected function get_matcher_to_query_var_map() {
 	protected function get_localized_matchers() {
 		$localized_matchers = parent::get_localized_matchers();
 
+		// If possible add a `localized_slug` entry to each localized matcher to support multi-language.
+		array_walk( $localized_matchers, function ( array &$localized_matcher ) {
```

Comment:
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: 14313377399e13cae7e2d04e9d6c2f16</sub>